app/furniture/marketplace/breadcrumbs.rb

```diff
@@ -27,6 +27,23 @@
   link t("marketplace.orders.index.link_to"), marketplace.location(child: :orders)
 end
 
+crumb :marketplace_order_notification_methods do |marketplace|
+  parent :edit_marketplace, marketplace
+  link t("marketplace.order_notification_methods.index.link_to"), marketplace.location(child: :order_notification_methods)
+end
+
+crumb :new_marketplace_order_notification_method do |order_notification_method|
+  parent :marketplace_order_notification_methods, order_notification_method.marketplace
+  link t("marketplace.order_notification_methods.index.link_to"),
+    order_notification_method.marketplace.location(child: :order_notification_methods)
+end
+
+crumb :edit_marketplace_order_notification_method do |order_notification_method|
+  parent :marketplace_order_notification_methods, order_notification_method.marketplace
+  link t("marketplace.order_notification_methods.edit.link_to", contact_location: order_notification_method.contact_location)
+  order_notification_method.marketplace.location(child: :order_notification_methods)
+end
```

Comment:
One day we will not have to define crumbs because we have made them so consistent that we can just infer wht the crumb should be...

ONE DAY....